snuba/query_engine.py

```diff
@@ -0,0 +1,201 @@
+import logging
+
+from clickhouse_driver.errors import Error as ClickHouseError
+from collections import namedtuple
+from hashlib import md5
+from typing import Any, MutableMapping, NamedTuple
+
+from snuba import settings, state
+from snuba.clickhouse.native import ClickhousePool
+from snuba.request import Request
+from snuba.util import (
+    all_referenced_columns,
+    create_metrics,
+    force_bytes,
+    Timer,
+    to_list,
+)
+
+logger = logging.getLogger('snuba.query_engine')
+metrics = create_metrics(settings.DOGSTATSD_HOST, settings.DOGSTATSD_PORT, 'snuba.api')
```

Comment:
I don't think `snuba.util.metrics` is directly referenced/imorted anywhere: I think it was just a used in `raw_query` in place of a local variable. I am pretty sure the `metrics` assignment in `snuba.util` is safe to remove as long as it was moved here.

Reply:
done